lexer.py

```
@@ -0,0 +1,100 @@
+from dataclasses import dataclass
+from enum import Enum, auto
+from typing import Callable
+
+
+class TokenType( Enum ):
```

Flake8 (7.2.0)

[error] whitespace after '('

(E201)